resources/[basketball]/rcore_basketball/config.lua

```diff
@@ -22,7 +22,7 @@ Config = {
 		Flat = false,
 	},
 
-	HideMarkers = false,
+	HideMarkers = true,
```

Comment:
I think those markers are probably pretty useful to have on
```suggestion
	HideMarkers = false,
```